htmresearch/frameworks/nlp/imbu.py

```diff
@@ -327,7 +353,7 @@ def main():
                            "header 'Sample'; see readCSV() for more.",
                       required=True)
   parser.add_argument("-m", "--modelName",
-                      choices=list(ClassificationModelTypes.getTypes()),
+                      choices=ImbuModels.modelMappings.keys(),
```

Comment:
Need not use `.keys()`, here.  `choices=ImbuModels.modelMappings` will use the keys by nature of iterating over `dict`s in python.